Atomic/atomic.py

```diff
@@ -134,7 +134,7 @@ class Atomic(object):
                 "--name", "${NAME}",
                 "${IMAGE}"]
 
-    def __init__(self):
+    def __init__(self, is_dbus = False):
         self.d = get_docker_client()
```

Comment:
Not sure I follow you here @rhatdan .  it will be nice to know if things are coming in on dbus in the code.  He could also do:

```
if self.is_dbus:
     self.is_tty = True
```